docs/design/storagenode-downtime-tracking.md

```diff
@@ -2,29 +2,27 @@
 
 ## Abstract
 
-The goal is tracking Storage Nodes downtime per Satellite.
+This document describes Storage Node downtime tracking.
 
 ## Background
 
-The [Disqualification design document](disqualification.md) describes when the Storage Nodes get disqualified based on the reputation scores described in the [Node Selection design document](node-selection.md).
+[Disqualification design document](disqualification.md) describes how Storage Nodes get disqualified based on the reputation scores described in [node selection design document](node-selection.md).
 
-The current disqualification showed to us that several Storage Nodes got disqualified because of their uptime stats without giving clear and fair evidence of it to us nor their operators. We had to take action and invalidate the uptime's part of the formula used for calculating their reputation.
+Current disqualification approach disqualified several nodes, because of their uptime stats, without clear and fair evidence. These disqualifications needed to be reverted and the disqualification system disabled. Before we can start handling disqualifications we need to more reliably track offline status of nodes.
```

Comment:
We haven't disabled the disqualification entirely, we disabled only the uptime disqualification part.

And it isn't actually disabled is stilly apply but it doesn't impact the total score.

Reply:
If disables look good for you it's fine, but technically isn't disabled, it's actually calculated but its result is always 0.